Makefile

```
@@ -73,26 +74,35 @@ ifeq ($(MAKECMDGOALS),statusgo-ios-library)
     CGO_LDFLAGS+=-Os -flto
 endif
 
+CGO_CFLAGS = -I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
+export GOPATH ?= $(HOME)/go
+
+GIT_ROOT ?= $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+GIT_COMMIT ?= $(shell git rev-parse --short HEAD)
+GIT_AUTHOR ?= $(shell git config user.email || echo $$USER)
+
+export NIM_SDS_HEADER_PATH = $(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/library
+export NIM_SDS_LIB_PATH = $(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/build
```

We should probably allow overrides too right?

```suggestion
export NIM_SDS_HEADER_PATH ?= $(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/library
export NIM_SDS_LIB_PATH ?= $(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/build
```

---

Solved in 8d09807720b0ec1011a8ffcbac6a93ff9dca5337 and https://github.com/status-im/status-go/pull/6884/commits/d96d74ad9df958d1760211839ec866b5143a21c7